link to documentation to the com1DFA_Exe repository (#163)
avaframe/AvaFrame: docs/moduleCom1DFA.rst

To run
--------

> * the computation of the com1DFA module relies on the samos-AT C++ code.

Reviewer: ```suggestion
* the computation of the com1DFA module relies on a com1DFA C++ executable.
```

>   The executable (for now only the linux one is available but a win64 will be soon added) and attribute files are

Reviewer: ```suggestion
  The executable (for now only a 64bit linux one is available) and needed files are
```

  available in this `git repository <https://github.com/avaframe/com1DFA_Exe>`_.

  .. Hint:: Simply clone the `com1DFA_Exe <https://github.com/avaframe/com1DFA_Exe>`_ git repository.

* create an avalanche directory with required input files - for this task you can use :ref:`moduleIn3Utils:Initialize Project`
* copy ``avaframeCfg.ini`` to ``local_avaframeCfg.ini`` and set your desired avalanche directory name
* copy ``com1DFACfg.ini`` to ``local_com1DFACfg.ini`` and set your desired settings. Make sure you update properly

>   the path to samos-AT C++ executable and attribute files (the one you just cloned).

Reviewer: ```suggestion
  the path to com1DFA C++ executable and attribute files (the one you just cloned).
```

>   .. Attention:: Make sure you update properly the path to samos-AT C++ executable

Reviewer: ```suggestion
  .. Attention:: Make sure you update the path to com1DFA C++ executable
```
